### PR TITLE
Updated motion controllers to use final GLB assets.

### PR DESCRIPTION
**Pull request by leweaver**
Fixed pointing pose on motion controllers to read from the mesh

### CODE REVIEW

File: src/Gamepad/Controllers/babylon.windowsMotionController.ts [hunk 1]
```diff
@@ -24,10 +23,9 @@ module BABYLON {
     }
 
     export class WindowsMotionController extends WebVRController {
-        private static readonly MODEL_BASE_URL:string = 'https://controllers.babylonjs.com/microsoft/';
+        private static readonly MODEL_BASE_URL:string = '/assets/meshes/controllers/microsoft/';
```
Comment (deltakosh) [inline src/Gamepad/Controllers/babylon.windowsMotionController.ts]: This looks suspicious :)

File: src/Gamepad/Controllers/babylon.windowsMotionController.ts [hunk 2]
```diff
@@ -247,15 +246,16 @@ module BABYLON {
             let childMesh : AbstractMesh = null;
             for (let i = 0; i < meshes.length; i++) {
                 let mesh = meshes[i];
-                if (mesh.id === WindowsMotionController.MODEL_ROOT_NODE_NAME) {
-                    // There may be a parent mesh to perform the RH to LH matrix transform.
+
+                // There may be a parent mesh to perform the RH to LH matrix transform.
+                if (mesh.parent && mesh.parent.name === WindowsMotionController.GLTF_ROOT_TRANSFORM_NAME)
```
Comment (deltakosh) [inline src/Gamepad/Controllers/babylon.windowsMotionController.ts]: The parent is now returned as part of the meshes list

Comment (leweaver) [reply src/Gamepad/Controllers/babylon.windowsMotionController.ts]: Ok - so does that mean we are guaranteed to find a mesh with a null parent? In which case I can drop the .name check altogether

File: src/Gamepad/Controllers/babylon.windowsMotionController.ts (hunk 1 above)
Comment (leweaver) [reply src/Gamepad/Controllers/babylon.windowsMotionController.ts]: Yeh accidental commit sorry - removed!

File: src/Gamepad/Controllers/babylon.windowsMotionController.ts (hunk 2 above)
Comment (leweaver) [reply src/Gamepad/Controllers/babylon.windowsMotionController.ts]: OK I updated this section, no longer testing for 'root' and it is working great :)

**Comment by deltakosh** [comment]
Perfect! thanks!

**Pull request merged**